roles/ansible-runner/files/usr/local/bin/ansible-runner

```diff
@@ -11,29 +11,33 @@ envs_root=${ANSIBLE_RUNNER_ENV_ROOT:-/opt/source}
 ansible_venv=${ANSIBLE_RUNNER_VENV:-/opt/ansible}
 log_dir=${ANSIBLE_RUNNER_LOG_DIR:-/var/www/html/cron-logs/$env/}
 ssh_user=${ANSIBLE_SSH_USER:-""}
+force_runner=false
+ansible_options=""
 
 if ! ([ -n "$ANSIBLE_PLAYBOOK" ] && [ -n "$ANSIBLE_INVENTORY" ]); then
   echo "Missing all required environment variables: ANSIBLE_PLAYBOOK, ANSIBLE_INVENTORY"
   exit 1
 fi
 
-if [[ "$*" == *"--force"* && "$*" != *"--force-"* ]]; then
-    echo "--force detected, ignoring /etc/disable-ansible-runner-$env..."
+for argument in "$@"; do
+    if [[ "$argument" == "--force-runner" ]]; then
+        force_runner=true
+    else
+        ansible_options="$ansible_options $argument"
+    fi
+    shift
```

Comment:
This 'shift' is not needed since we're not actually handling positional arguments /w the numerical variables at this point ($1, $2, etc)